PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -664,7 +666,7 @@ void WindowsDlg::updateColumnNames()
 
 void WindowsDlg::onSize(UINT nType, int cx, int cy)
 {
-	MyBaseClass::onSize(nType, cx, cy);
+	MyBaseClass::onSize(nType, cx, cy);					

```

Comment:
Remove all the space characters after `MyBaseClass::onSize(nType, cx, cy);`.

Reply:
Removed